framework/base/src/contract_base/wrappers/send_wrapper.rs

```diff
@@ -358,16 +358,6 @@ where
             .async_call_and_exit()
     }
 
-    /// Creates a call to the `ClaimDeveloperRewards` builtin function.
-    ///
-    /// In itself, this does nothing. You need to then call turn the contract call into an async call.
-    pub fn claim_developer_rewards(
```

Comment:
Do not delete methods in the SendWrapper!

Simply change their return type and implementation.